abstract_specification.md

```diff
@@ -1,26 +1,29 @@
 # Introduction
 
-> **Status**: in active development. Version: `0.2.0-dev`
->
-> ℹ️ **Note** - this is an early working draft (WORK IN PROGRESS). It may contain controversial, undecided, or incomplete proposals - feedback is encouraged. It is expect that additional revisions will be required to converge the underlying primitives with use cases, and to incorporate lessons from initial implementation and early users. Please see `issues` and other call-outs throughout the doc for a list of active discussions/debates.
+<!-- Note to editors: this version string is independent of GitHub release tags on this repo. In particular, it needs to match `get_SOMA_version()` in impls. -->
+
+**Specification version**: `0.2.0-dev`
+
+ℹ️ **Note**: Feedback on this spec is encouraged. Please [file an issue](https://github.com/single-cell-data/SOMA/issues)
+with any and all feedback, comments, or concerns.
 
 The goal of SOMA (“stack of matrices, annotated”) is a flexible, extensible, and open-source API providing access to annotated, 2D matrix data stored in multiple underlying formats and systems. The vision for this API family includes:
 
-- support access to persistent, cloud-resident datasets,
-- enable use within popular data science environments (e.g., R, Python), using the tools of that environment (e.g., Python Pandas integration),
+- support access to persistent, cloud-resident datasets
```

Comment:
do we want periods on bullet points or not? they're present on some and absent on others.

Reply:
I've been very explicitly trained to put the period at the end of the "sentence" -- which these are -- since another sentence follows them. Likewise with `,` or `;` between bullets.

I've already let go of the latter, on your insistence, to move this PR forward.

It'd be quickest if you just threw down your preference here & I'll bulk-edit to match.

Reply:
My preference was to treat them as independent portions of a sentence, as in the example:

Our goals are to:

- eat lunch.
- construct a dangerous contraption.
- yell at passers by to make them uncomfortable.

Reply:
@thetorpedodog will do.